### PR TITLE
config: add non-main files checkstyle run

### PR DESCRIPTION
**Pull request by rnveach**
Identified at https://github.com/sevntu-checkstyle/sevntu.checkstyle/issues/943 when looking at the logs which were full of download messages.

### CODE REVIEW

File: .ci/bump-cs-version-in-esclipsecs-plugin.sh [hunk 1]
```diff
@@ -10,6 +10,6 @@ fi
 VERSION=$1
 cd eclipsecs-sevntu-plugin
 
-mvn versions:set-property -DgenerateBackupPoms=false -Dproperty=checkstyle.version -DnewVersion="$VERSION"
+mvn -e --no-transfer-progress versions:set-property -DgenerateBackupPoms=false -Dproperty=checkstyle.version -DnewVersion="$VERSION"
```
Comment (rnveach) [inline .ci/bump-cs-version-in-esclipsecs-plugin.sh:13]: @stoyanK7 ping

File: sevntu-checks/config/ant-phase-verify.xml [hunk 2]
```diff
@@ -42,6 +42,73 @@
...
+    </tstamp>
+    <echo level="info">Checkstyle started (${checkstyle.nonMain.configLocation}): ${STARTED}</echo>
+
+    <checkstyle config="${checkstyle.nonMain.configLocation}"
+                failOnViolation="true"
+                failureProperty="checkstyle.failure.property"
+                executeIgnoredModules="true"
+                >
+      <path>
+        <fileset dir="${basedir}/.." includes="**/*">
```
Comment (rnveach) [inline sevntu-checks/config/ant-phase-verify.xml:56]: This is one of the main differences from main repo. This run is in a sub-directory, so we have to go up one to collect the entire repo. This also means a lot of the excludes have to be adjusted and that is why they start with "**/".

File: sevntu-checks/config/checkstyle_non_main_files_suppressions.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+    "-//Checkstyle//DTD SuppressionFilter Configuration 1.1//EN"
+    "https://checkstyle.org/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+  <!-- START of legacy code, all violations will be resolved -->
+  <suppress id="lineLength" files=".*"/>
```
Comment (rnveach) [inline sevntu-checks/config/checkstyle_non_main_files_suppressions.xml:9]: This is legacy as it includes a lot of input files and such.

File: .ci/bump-cs-version-in-esclipsecs-plugin.sh (hunk 1 above)
Comment (stoyanK7) [reply .ci/bump-cs-version-in-esclipsecs-plugin.sh:13]: Noted.

**Review by romani — APPROVED** [review]

**Pull request merged**